_search-plugins/search-pipelines/ml-inference-search-response.md

```diff
@@ -565,17 +568,14 @@ The response will include the original documents and the generated summary in th
   }
 }
 ```
-{% include copy-curl.html %}
-This example demonstrates how the ml_inference search response processor can be used with a generative AI model to provide summarization of search results. The summary is included in the ext field of the search response, allowing for easy access to the AI-generated insights alongside the original search results.
 
-
-### Example: Rerank use case
+### Example: Reranking search results using a text similarity model
 
 The following example shows you how to configure an `ml_inference` search response processor with a text similarity model.
 
+**Prerequisite**
 
-Step 0: Host a model
-The pre-requisite is a registered text similarity model in OpenSearch. For more information about externally hosted models, see [Connecting to externally hosted models]({{site.url}}{{site.baseurl}}/ml-commons-plugin/remote-models/index/). Here is a sample predict, which requires a text and a text_pair field within inputs field. 
+You must configure an externally hosted LLM for this use case. For more information about externally hosted models, see [Connecting to externally hosted models]({{site.url}}{{site.baseurl}}/ml-commons-plugin/remote-models/index/). Once you register the LLM, you can use the following request to test it. This request requires providing a `text` and `text_pair` fields within the `inputs` field:
```

Comment:
@kolchfa-aws  this rerank use case is using a registered text similarity model not llm model